### DISCUSSION TITLE
创立者工业产品集中反馈帖

### DISCUSSION THREAD

**Discussion by GrunderIndustries** [share121/inter-knot#3120] [其他]
![屏幕截图 2024-08-18 233804](https://image.gcores.com/f0431133-53b8-4e4e-8b0a-5b20fbe97d92.png?x-oss-process=image/quality,q_90/format,webp/watermark,image_d2F0ZXJtYXJrLnBuZw,g_se,x_10,y_10)
# 任何有关创立者工业产品的改进意见和需求均可发在该贴下方
## 反馈时需将反馈产品名称贴出
# 此帖下方也可以发送招标信息

**Comment by LT-UT** [comment]
希望创立者工业的导弹产品可以支持导弹发射服务<img src="https://bbs-static.miyoushe.com/static/2023/11/16/810dd11a4d16759f16ca078596aab6b1_422007669816866994.png" alt="哲-让我想想" width="75" >

**Comment by warning-Kirov-reporting** [comment]
这是什么东西<img src="https://upload-bbs.miyoushe.com/upload/2022/12/21/496b41d101eec4c792e82da91fc29daf_2136519103324822738.png" alt="时光交响之章表情包17" width="75" >

**Reply by LT-UT** [reply]
哪个
